src/streaming/controllers/MediaController.js

```diff
@@ -89,7 +89,7 @@ function MediaController() {
             setTrack(selectInitialTrack(type, tracksForType), true);
         } else {
             if (tracks.length > 1) {
-                setTrack(selectInitialTrack(type, tracks, !!lastSelectedTracks[type]));
+                setTrack(selectInitialTrack(type, tracks));
```

Comment:
`selectInitialTrack` has only 2 arguments, maybe it's for other purposes, but it doesn't take any effects at the moment.